.travis.yml

```diff
@@ -1,11 +1,15 @@
 ---
 language: python
 
+env:
+  - SUBDIR=node
```

Comment:
There's not a great story for solving node version skew afaik, we baked our own internally to solve this problem. +@raynos for feedback on how this is usually dealt with wrt travis.